src/app/containers/ArticleMain/__snapshots__/index.test.jsx.snap

```diff
@@ -208,7 +208,7 @@ exports[`ArticleMain should render a news article correctly 1`] = `
       rel="amphtml"
     />
     <link
-      href="/news/images/icons/icon-192x192.png"
+      href="http://localhost.bbc.com:7080/news/images/icons/icon-192x192.png"
```

Comment:
These have changed as we are now have `SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN` and `SIMORGH_PUBLIC_STATIC_ASSETS_PATH` setup in Jest config - https://github.com/bbc/simorgh/pull/4117/files#diff-77cca5a3ad8587ce98baf1f2d6534917R26-R29